manifests/init.pp

```diff
@@ -99,19 +102,23 @@
   Variant[Boolean[false], Pattern[/icmp(v6|x)? type .+|tcp reset/]] $reject_with = 'icmpx type port-unreachable',
   Variant[Boolean[false], Enum['mask']] $firewalld_enable = 'mask',
   Optional[Array[Pattern[/^(ip|ip6|inet)-[-a-zA-Z0-9_]+$/],1]] $noflush_tables = undef,
+  Stdlib::AbsolutePath $nftables_configuration_path = '/etc/sysconfig/nftables.conf',
```

Comment:
I think just `configugration_path` is enough as a variable name.

Reply:
Sounds good, I've made that change